lib/rules/lint-no-static-element-interactions.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const AstNodeInfo = require('../helpers/ast-node-info');
+const Rule = require('./base');
+
+const errorMessage = 'Static HTML elements with event handlers require a role.';
+
+// const COMMON_INTERACTIVE_ROLES = [
+//   'button',
+//   'link',
+//   'checkbox',
+//   'menuitem',
+//   'menuitemcheckbox',
+//   'menuitemradio',
+//   'option',
+//   'radio',
+//   'searchbox',
+//   'switch',
+//   'textbox',
+// ];
+
+// role || aria-
+
+const handlers = ['onclick', 'onmousedown', 'onmouseup', 'onkeypress', 'onkeydown', 'onkeyup'];
+const actionHandlers = ['click', 'mouseDown', 'mouseUp', 'keyPress', 'keyDown', 'keyUp'];
+const NOT_INTERACTIVE_ELEMENTS = [
```

Comment:
Hmm, seems like we should be reusing the interactive utility methods instead of making more.

See https://github.com/ember-template-lint/ember-template-lint/blob/master/lib/helpers/is-interactive-element.js for details

Reply:
Done